__tests__/notifications.ts

```diff
@@ -354,43 +354,15 @@ describe('query notificationPreferences', () => {
       'UNAUTHENTICATED',
     ));
 
-  const prepareNotificationPreferences = async () => {
```

Comment:
Just a duplicated code that happened through some merging. There should be another one a bit above this code block.